### ISSUE TITLE
Not able to backup, unexpected token

### ISSUE THREAD

**Issue by thibmaek** [dfrankland/hyper-sync-settings#9]
``` console
VM398:7 Uncaught SyntaxError: Unexpected token .
electron.ipcRenderer.on @/Applications/HyperTerm.app/Contents/Resources/electron.asar/renderer/init.js:40
emitMany @ events.js:127
emit @ events.js:201
```

``` js
(
      () => {
        const notification = new Notification(
          'hyperterm-sync-settings error 🔥',
          {
            body: 'Error: EINVAL: invalid argument, readlink '/Users/username/.hyperterm_plugins/.hyperterm-sync-settings/.hyperterm.js'',
          }
        );
        undefined
      }
    )();
```

**Comment by dfrankland** [comment]
Oh!

As far as the `Error: EINVAL`... error goes, that seems to be an issue with accessing the file. What OS are you running HyperTerm on?

There seems to be several problems here with the `Notification`, that are separate from the file access issue too:
1. [This line](https://github.com/dfrankland/hyperterm-sync-settings/blob/master/lib/getOpen.js#L25) should return an empty string if `url` is falsey, not `undefined`.
2. [This line](https://github.com/dfrankland/hyperterm-sync-settings/blob/master/lib/getOpen.js#L22) should escape single quotes.

**Comment by thibmaek** [comment]
Hmm this seems to be resolved using the workaround from #8 
Running on OS X latest.

**Comment by dfrankland** [comment]
If you get another issue, feel free to create another issue. Closing for now.
